docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/install-and-configure-stripe-prerequisites.md

```diff
@@ -217,6 +217,102 @@ use Spryker\Zed\OauthClient\Communication\Plugin\Payment\AccessTokenPaymentAutho
 
 ```
 
+## Using Stripe in Marketplace context
+
+Install the required modules:
+
+```bash
+composer require spryker/merchant-app spryker/merchant-app-merchant-portal-gui
+```
+
+### Configuration
+
+To enable your application to work with Stripe in a Marketplace context, you need to configure your application by adding some plugins, update some configurations, and update your state-machine configuration.
+
+#### config_default.php
+
+Update your `config_default.php` file with the following changes:
+
+```php
+$config[OauthClientConstants::OAUTH_PROVIDER_NAME_FOR_ACP]
+$config[OauthClientConstants::OAUTH_GRANT_TYPE_FOR_ACP]
+$config[OauthClientConstants::OAUTH_OPTION_AUDIENCE_FOR_ACP]
+$config[KernelAppConstants::TENANT_IDENTIFIER]
+```
+
+`OauthClientConstants` are replacing the deprecated `OauthClientConstants::OAUTH_PROVIDER_NAME_FOR_PAYMENT_*` constants.
+
+#### Navigation
+
+Update `config/Zed/navigation.xml` as follows:
+
+```xml
+<?xml version="1.0"?>
+<config>
+    ...
+
+    <merchant-portal-payment-settings>
+        <label>Payment Settings</label>
+        <title>Payment Settings</title>
+        <icon>payment</icon>
+        <bundle>merchant-app-merchant-portal-gui</bundle>
+        <controller>payment-settings</controller>
+        <action>index</action>
+        <pages>
+            <onboarding>
+                <label>Onboarding</label>
+                <title>Onboarding</title>
+                <icon>payment</icon>
+                <bundle>merchant-app-merchant-portal-gui</bundle>
+                <controller>payment-settings</controller>
+                <action>onboarding</action>
+                <visible>0</visible>
+            </onboarding>
+        </pages>
+    </merchant-portal-payment-settings>
+</config>
+```
+
+#### Plugins
+
+##### KernelApp
+- `\Spryker\Zed\MerchantApp\Communication\Plugin\KernelApp\MerchantAppRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
```

Comment:
What type of entity is `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`?

Reply:
This is just the method that loads all request expanders. The expander are then used to add additional data to the request. In this case the merchant reference will be added to the request before it gets send to ACP.